orc8r/cloud/go/services/orchestrator/obsidian/models/conversion.go

```diff
@@ -75,6 +80,14 @@ func (m *NetworkFeatures) ToUpdateCriteria(network configurator.Network) (config
 	return GetNetworkConfigUpdateCriteria(network.ID, orc8r.NetworkFeaturesConfig, m), nil
 }
 
+func (m *NetworkSentryConfig) GetFromNetwork(network configurator.Network) interface{} {
+	return GetNetworkConfig(network, orc8r.NetworkSentryConfig)
+}
+
+func (m *NetworkSentryConfig) ToUpdateCriteria(network configurator.Network) (configurator.NetworkUpdateCriteria, error) {
+	return GetNetworkConfigUpdateCriteria(network.ID, orc8r.NetworkSentryConfig, m), nil
```

Comment:
These lines should be indirectly covered through the tests in [network_handlers_test.go](https://github.com/magma/magma/pull/7638/files#diff-1e3584f3517597a3aa63e19a1e9c69b3f04fd065b144d6d61543353e8330dd25R164).